src/implementations/wrap/wrap.ts

```diff
@@ -160,11 +166,24 @@ const loadSchema = (schemaPath?: string): Promise<Schema | undefined> => {
 
 export const wrap = async (
```

Comment:
let's refactor this, move everything into the option object (inputPath and outputPath)

Reply:
done